MobalMobal/MobalMobal/Main/cell/MainMyDonationTableViewCell.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  MainMyDonationTableViewCell.swift
+//  MobalMobal
+//
+//  Created by 김재희 on 2021/02/28.
+//
+
+import SnapKit
+import UIKit
+
+class MainMyDonationTableViewCell: UITableViewCell {
+    // MARK: - UIComponent
+    let collectionView: UICollectionView = {
+        let collectionView: UICollectionView = UICollectionView(frame: .zero, collectionViewLayout: UICollectionViewFlowLayout())
+        let layout: UICollectionViewFlowLayout = UICollectionViewFlowLayout()
+        layout.scrollDirection = .horizontal
+        collectionView.collectionViewLayout = layout
```

Comment:
FlowLayout을 먼저 선언하고, collectionView를 선언하면서 초기화 할 때 flowLayout을 넣어주는 건 어떨까요?
주요 컴포넌트인 collectionView를 먼저 선언하는게 좋다고 생각하면 먼저 해도 될 것 같긴 합니다. 취향 차이인 듯!

```suggestion
        let layout: UICollectionViewFlowLayout = UICollectionViewFlowLayout()
        layout.scrollDirection = .horizontal
        let collectionView: UICollectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
```